src/cephadm/cephadmlib/node_proxy/requirements.txt

```diff
@@ -0,0 +1,7 @@
+pyyaml
+types-PyYAML
+types-requests
+mypy
+flake8
+redfish
+CherryPy
```

Comment:
I assume these aren't actually connected to build.py, given I see some deps in here (redfish, cherrypy) that I think you dropped already.

Reply:
This `requirements.txt` is actually a left over from before I migrate the code to the cephadm agent.